src/FilamentManager.php

```diff
@@ -55,13 +89,37 @@ public function registerRole($role)
         $this->roles = array_merge($this->roles, [$role]);
     }
 
+    public function registerScript($name, $script)
+    {
+        $this->scripts[$name] = $script;
```

Comment:
```suggestion
        $this->scripts[$name] = array_merge($this->scripts, [$path]);
```

Reply:
@danharrin This isn't right. The point of the `$name` variable is so that each script / style registered has a unique name and the same `$name` can't be present more than once (allows you to override vendor scripts in an extension package for example).